### DISCUSSION TITLE
SX1262 question.

### DISCUSSION THREAD

**Discussion by EliasA97** [jgromes/RadioLib#568] [General]
Hello, what's the range of there values (bandwidth, spreading factor, coding rate)? Also are there any specific values that can we choose from?

**Comment by jgromes** [comment]
I would suggest checking the SX1262 datasheet, or the online RadioLib API reference: https://jgromes.github.io/RadioLib/class_s_x126x.html#a2f60df59c80241d98ce078c0417a7f08

**Comment by EliasA97** [comment]
Thanks, that was enlightening. What about coding rate? I can't find it on the datasheet. What does it?

**Reply by jgromes** [reply]
Coding rate is the amount of fordward error correction (FEC) the LoRa modem performs, higher number means more FEC, therefore higher link budget. However, it also creates longer packets, so longer time-on-air and higher power consumption. I'm pretty sure it's in the datasheet ;)

**Comment by EliasA97** [comment]
You're right, it was there in front of me!!
Thank's!
